libkbfs/tlf_journal.go

```diff
@@ -1170,6 +1170,12 @@ func (j *tlfJournal) removeFlushedMDEntry(ctx context.Context,
 func (j *tlfJournal) flushOneMDOp(
 	ctx context.Context, end MetadataRevision,
 	maxMDRevToFlush MetadataRevision) (flushed bool, err error) {
+	if maxMDRevToFlush == MetadataRevisionUninitialized {
+		// Short-cut `getNextMDEntryToFlush`, which would otherwise read
+		// an MD from disk and sign it unnecessarily.
```

Comment:
> We can probably add a method to the MD journal to get the earliest ordinal (== revision) and also use that too. Maybe add a TODO?

Why would that be better?  Seems to me that everything we need to know is already in `maxMDRevToFlush`, no need to take the journal lock to figure out redundant info here.

> Also it's a bit weird to have both maxMDRevToFlush and end

Yeah, you're right, it's largely historical, though I guess it could help if the block and md journals get out of sync somehow.  I'll add a TODO/comment.